framework/widgets/ActiveField.php

```diff
@@ -775,19 +775,19 @@ public function widget($class, $config = [])
         $config['attribute'] = $this->attribute;
         $config['view'] = $this->form->getView();
         if (is_subclass_of($class, 'yii\widgets\InputWidget')) {
-            foreach ($this->inputOptions as $key => $value) {
-                if (!isset($config['options'][$key])) {
-                    $config['options'][$key] = $value;
-                }
-            }
+            $classOptions = (new \ReflectionClass($class))->getDefaultProperties();
+            $classOptions = is_null($classOptions['options']) ? [] : $classOptions['options'];
```

Comment:
```suggestion
            $classOptions = $classOptions['options'] === null ? [] : $classOptions['options'];
```

Reply:
Guessing this change is suggested to (micro)optimize performance? Or something else?

Reply:
Just the code style. Nothing too important.